egs2/mls/asr2/RESULTS.md

```diff
@@ -0,0 +1,49 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Wed Sep 13 18:26:42 EDT 2023`
+- python version: `3.8.6 (default, Dec 17 2020, 16:57:01)  [GCC 10.2.0]`
+- espnet version: `espnet 202308`
+- pytorch version: `pytorch 1.13.1+cu117`
+- Git hash: `a8bc43b1bfc9518da7dd8be4cad0ef346ef222fc`
+  - Commit date: `Sun Aug 20 16:17:23 2023 -0400`
+
+## exp/asr_smallerbatch_wamrup10k_lr0.0001_e200_raw_wavlm_large_21_full_km1000_bpe_rm3000_bpe_ts150
```

Comment:
You may consider put the model to huggingface as well

Reply:
Uploaded to huggingface (5584524), thx for the tip!